app/services/builders/metric_chart/blog/growth_month_over_month.rb

```diff
@@ -0,0 +1,26 @@
+module Builders
+  module MetricChart
+    module Blog
+      class GrowthMonthOverMonth < Builders::MetricChart::Blog::Base
+        def initialize(months = 13)
+          super(months + 1)
+        end
+
+        private
+
+        def process_data(metrics_data)
+          metrics_data.each_with_object({}) do |(month, metric_value), hash|
+            previous_metric_value = metrics_data[month.last_month]
```

Comment:
just to check: is `month.last_month` the previous month of the one in the iteration?

Reply:
Correct!